guides/source/active_job_basics.md

```diff
@@ -330,6 +330,31 @@ class GuestsCleanupJob < ActiveJob::Base
 end
 ```
 
+### Deserialization
+
+GlobalID, which Active Job uses, allows serializing full ActiveRecord objects as an argument to `#perform`.
+
+```ruby
+def perform(user)
+  user.send_welcome_email!
+end
+```
+
+if the `User` record is deleted after the job is enqueued but before the perform method is called
```

Comment:
Then you can write something like:

> If a passed record is deleted after ...

Also could you put backticks around "perform" ?